src/App.vue

```diff
@@ -289,12 +289,13 @@ export default {
       this.dataLoaded = false
 
       // store today's date every time the dashboard is loaded
-      const serverDate = await this.getServerDate()
+      const jsDate: Date = await this.getServerDate()
       if (!this.isJestRunning) {
         // eslint-disable-next-line no-console
-        console.info(`It is currently ${this.dateToPacificDateTime(serverDate)}.`)
+        console.info(`It is currently ${this.dateToPacificDateTime(jsDate)}.`)
       }
-      this.setCurrentDate(this.dateToYyyyMmDd(serverDate))
+      this.setCurrentJsDate(jsDate)
```

Comment:
Now we store the Date object from the server. It is used in the Date Mixin as a source of truth for creating other dates. (This way we don't need to trust that the user's browser/OS has the correct timezone to convert between UTC and Pacific.)